authorization/templates/authorization-service.yaml

```diff
@@ -62,7 +62,7 @@ metadata:
   annotations:
     kompose.cmd: kompose convert
     kompose.version: 1.19.0 (f63a961c)
-  name: dwv-web-{{ .Values.authorizationService.meta.name }}
+  name: dwv-web-{{ .Values.authorizationService.meta.name }}-old
```

Comment:
what's this used for? does the `-old` mean its deprecated?

Reply:
it can be removed now. ill do that

Reply:
next commit